table/column.go

```diff
@@ -77,12 +77,17 @@ func ToColumn(col *model.ColumnInfo) *Column {
 }
 
 // FindCols finds columns in cols by names.
-func FindCols(cols []*Column, names []string) ([]*Column, error) {
+func FindCols(cols []*Column, names []string, pkIsHandle bool) ([]*Column, error) {
 	var rcols []*Column
 	for _, name := range names {
 		col := FindCol(cols, name)
 		if col != nil {
 			rcols = append(rcols, col)
+		} else if name == model.ExtraHandleName.L && !pkIsHandle {
```

Comment:
Could we new a Column object in the Table meta? So we could avoid creating new object here.

Reply:
This is not always created, only if the user explicitly specify `_tidb_rowid`.